src/t2iapi/device/types.proto

```diff
@@ -80,3 +82,26 @@ enum CalibrationState{
     CALIBRATION_STATE_OTHER = 4;
 }
 
+
+/*
+Contains the meta data of the device.
+ */
+message MetaDataMsg {
```

Comment:
Why is this not in the biceps package, but all of the other types are? seems inconsistent to me